### PR TITLE
Enhancement/button states

### PR DESCRIPTION
**Pull request by cricar6**

### CODE REVIEW

**Review by javiermlovo — APPROVED** [review]

File: components/layouts/BigImageLayout.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div
+      className={
+        (variation !== 'none')
+          ? `big-image-layout big-image-layout--${variation}`
+          : 'big-image-layout'
+      }
+    >
+      <div className="big-image-layout__content">
+        { title
```
Comment (Cruzlex7) [inline components/layouts/BigImageLayout.js:24]: it could be 
```suggestion
        {title && (<h2> {title} </h2>)}
```

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import getPageData from '../utils/api';
+import BigImageLayout from '../components/layouts/BigImageLayout';
+import LastProjects from '../components/sections/LastProjects';
+import JoinUs from '../components/sections/JoinUs';
 
 export const getServerSideProps = async () => {
-  const pageData = await getPageData('home');
+  const pageData = await getPageData('inicio');
```
Comment (Cruzlex7) [inline pages/index.js:7]: must be a constat

File: pages/index.js [hunk 3]
```diff
@@ -12,14 +15,40 @@ export const getServerSideProps = async () => {
...
     <div className="home">
-      <h1>
-        Index page
-      </h1>
-      {components.map((component) => (
-        <p key={component.sys.id}>{component.fields.title}</p>
-      ))}
+      <JoinUs>
+        <BigImageLayout
+          title={joinUsSection.fields.title}
```
Comment (Cruzlex7) [inline pages/index.js:26]: provide only one object as a prop that contains al this info.

**Review by Cruzlex7 — CHANGES_REQUESTED** [review]
This PR must be merge with dev, not with main branch.

**Pull request closed without merging**